ios/gutenberg/GutenbergViewController.swift

```diff
@@ -70,9 +70,9 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
             print("Gutenberg did request media picker, passing a sample url in callback")
             switch currentFilter {
             case .image:
-                callback(1, "https://cldup.com/cXyG__fTLN.jpg")
+                callback([(id: 1, url: "https://cldup.com/cXyG__fTLN.jpg")])
             case .video:
-                callback(2, "https://i.cloudup.com/YtZFJbuQCE.mov")
+                callback([(2, "https://i.cloudup.com/YtZFJbuQCE.mov")])
```

Comment:
Looking inconsistent compared to line 73 since it has `id:`, `url:` labels for tuple items